configure.ac

```diff
@@ -124,6 +124,8 @@ AC_ARG_ENABLE([compress-all],
 	[AS_HELP_STRING([--disable-compress-all],[disable libknet all compress modules support])],,
 	[ enable_compress_all="yes" ])
 
+KNET_OPTION_DEFINES([zstd],[compress],[PKG_CHECK_MODULES([libzstd], [libzstd])])
+
```

Comment:
Hi, just wondering whether this empty line signifies that zstd is somehow different from the other compression plugins...